test/rules/structured-imports/ancestorCases.ts.lint

```diff
@@ -0,0 +1,11 @@
+import * as relative from "./relativeImport1";
+import { relativeImport } from "./relativeImport2";
+import { anotherRelativeImport } from "../ancestorImport";
+                                      ~~~~~~~~~~~~~~~~~~~~ [Imports should be listed in the following order: module imports, absolute imports, relative imports.]
```

Comment:
If you have a repeating error message, tslint's test runner allows you to define a footnote for it, and use that as shorthand.

* [Footnote definition](https://github.com/palantir/tslint/blob/master/test/rules/await-promise/test.ts.lint#L26)
* [Shorthand error message](https://github.com/palantir/tslint/blob/master/test/rules/await-promise/test.ts.lint#L11)

Less PR churn if a message changes.